.gitignore

```diff
@@ -2,3 +2,4 @@ dist
 bower_components
 node_modules
 .sizecache.json
+*.swp
```

Comment:
Please don't modify this file. You should add this to your own [`core.excludesfile`](https://help.github.com/articles/ignoring-files/).